src/components/boardComponent/boardComponent.js

```diff
@@ -2,29 +2,23 @@ import React, { Component } from 'react';
 import Square from '../squareComponent/squareComponent';
 import GameHeader from '../gameHeaderComponent/gameHeaderComponent';
 import './BoardComponent.css';
+import BoardView from '../../board/board';
```

Comment:
Why are you naming the imported class as `BoardView`? Aren't you importing the `Board` here?